types/office-js/index.d.ts

```diff
@@ -9632,6 +9824,54 @@ declare namespace Office {
         setAsync(headers: Object, callback?: (result: Office.AsyncResult<void>) => void): void;
     }
 
+    /**
+     * Represents a location. Read only.
+     * 
+     * [Api set: Mailbox Preview]
+     * 
+     * @beta
+     */
+    export interface LocationDetails {
+        /**
+         * The location's id.
+         */
+        locationId: LocationIdentifier;
+        /**
+         * The location's display name.
+         */
+        displayName: string;
+        /**
+         * The email address associated with the location.
+         */
+        emailAddress: string;
+        /**
+         * Additional properties for the location.
+         */
+        properties: any;
```

Comment:
We do not return this currently. Maybe we should not document it. Let's confirm from Petra as well and do the needful